data/language/fr-FR.yml

```diff
@@ -2183,3 +2183,4 @@ strings:
   2128: Fenêtré
   2129: Plein écran
   2130: Plein écran (nv. bureau)
+  2263: "{COLOUR WINDOW_2}/199"
```

Comment:
instead of hardcoding the character count just make it a parameter as text input windows have different limits.

Reply:
In the 3th commit I've changed it in the language files, but I'm pretty sure that limit in windows like: "changing town name", "scenario details", "scenario name", etc, is still hardcoded here:
https://github.com/OpenLoco/OpenLoco/blob/29d1deea7a3c7f5f652fc519bb4156aef29c1d02/src/OpenLoco/Ui/TextInput.cpp#L12-L19

Reply:
I was thinking about the change company name window that has a 31 character limit. Not sure how that integrates into this as I'm pretty sure the text input let's you enter way more then it truncates later in the process. Have a look into it but we can leave it for future if it looks complex.

Reply:
Agreed, the 199 is misleading since many other text entry fields don't actually accept 199 characters. @BartoszKedziorek any chance you can get the character limit to reflect the actual limit for that field, not the generic textbox entry limit? It'd probably involve a lot of work tbh, perhaps you could make a PR to enforce textbox length limits based on the field it's setting, and then this PR would be merged after it and you'd be able to get the correct lengths. Just an idea

Reply:
> any chance you can get the character limit to reflect the actual limit for that field, not the generic textbox entry limit?
 It'd probably involve a lot of work tbh, perhaps you could make a PR to enforce textbox length limits based on the field it's setting, and then this PR would be merged after it and you'd be able to get the correct lengths.

Yeah, sure. I'm going to make your proposed PR first and after it is merged, I'll commit necessary changes to this PR.